mandatory/2-logic-error.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
-function trimWord(word) {
-  return wordtrim();
+function trimWord (word) {
+  return trimWord();
```

Comment:
@laxmikckarki Hi, well done! 
In think in line 4 (function 1), it is better to use word.trim(), because you use the trim() method to trim the string. If you look at the tests below the exercises, it can give you a clue.